spec/TodoList.spec.js

```diff
@@ -135,10 +136,18 @@ describe("TodoList", () => {
     expect(todoList.showAll()).toEqual([])
   })
 
-  it("delete throws error if not found", () => {
+  it('delete throws error if not found', () => {
     // set up
 
     // execute, verify
-    expect(() => todoList.deleteBy(1)).toThrowError("Item not found")
+    expect(() => todoList.deleteBy(1)).toThrowError('Item not found')
+  })
+
+  it('should only show 20 chars of the ToDos if there is more than one ToDo', () => {
+    const maxChar = TodoList.showAll().substring(0, 20) + '.....'
+
+    expect(maxChar.toBe(TodoList.showAll().substring(0, 20) + '.....'))
```

Comment:
This is trying to use the `showAll` function on the `TodoList` class directly because you have a capital T; the class is called `TodoList`. You should call class functions from an instance of a class, not the class itself. E.g.:

```js
const todoList = new TodoList()
todoList.showAll()
```

A test should be constructed like this:

1. Run the thing you want to test the output of, and save that output into a variable
2. Check the output matches what you expect

Right now, you're putting the logic itself into the test.

```js
todoList.create("something that is way way longer than 20 characters in length")
todoList.create("this is fine")
const output = todoList.showAll()
// the output of showAll should be an array containing todo objects,
// each of which should have a text property that is no longer than 20 characters long

expect(output).toEqual([
  { id: 1, text: "something that is wa...", status: 'incomplete' },
  { id: 1, text: "this is fine", status: 'incomplete' }
])
```

Reply:
this is great feedback. I will go back and refactor my code. 
Thanks